### ISSUE TITLE
[playwright] allow passing selector to rules config

### ISSUE THREAD

**Issue by stefanprobst** [dequelabs/axe-core-npm#975]
### Product

playwright

### Feature Description

i am using [`react-aria-components`](https://react-spectrum.adobe.com/react-aria/index.html), which in their documentation suggest adding the following axe rules config to avoid false positives (see [here](https://react-spectrum.adobe.com/react-aria/accessibility.html#false-positives)):

```ts
{
  rules: [
    {
      id: 'aria-hidden-focus',
      selector: 'body *:not([data-a11y-ignore="aria-hidden-focus"])'
    }
  ]
}
```

this can for example be passed directly to the storybook a11y plugin, which under the hood passes it on to `axe.configure()` (see [here](https://github.com/storybookjs/storybook/blob/next/code/addons/a11y/src/a11yRunner.ts#L44)).

it would be cool if i could pass the same config directly to `AxeBuilder#options` from `@axe-core/playwright` as well. afaict this is currently not possible, because it only allows [enabling/disabling individual rules](https://github.com/dequelabs/axe-core-npm/blob/develop/packages/playwright/README.md#axebuilderoptionsoptions-axerunoptions), but not passing any additional rule config.

thanks!

**Comment by straker** [comment]
Thanks for the issue. This is not a feature we are planning on adding any time soon, but you can somewhat workaround the issue by passing a selector to `exclude` to completely ignore the element from accessibility checks.

**Comment by padmavemulapati** [comment]
@michael-siek can you please provide me QA-notes here

**Comment by padmavemulapati** [comment]
Closing it as QA-none
